docs/apache-airflow/authoring-and-scheduling/deferring.rst

```diff
@@ -141,6 +141,60 @@ The ``self.defer`` call raises the ``TaskDeferred`` exception, so it can work an
 
 ``execution_timeout`` on operators is determined from the *total runtime*, not individual executions between deferrals. This means that if ``execution_timeout`` is set, an operator can fail while it's deferred or while it's running after a deferral, even if it's only been resumed for a few seconds.
 
+Triggering Deferral from Start
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to defer your task directly to the triggerer without going into the worker, you can add the class level attributes ``start_trigger`` and ``_next_method`` to your deferrable operator.
```

Comment:
@Lee-W probably we should add a note here that this feature is available starting Airflow 2.10 ?
Similar to  https://github.com/apache/airflow/blob/54d01809f9389680b0ae358ee72bc162503dce3e/docs/apache-airflow/administration-and-deployment/logging-monitoring/logging-tasks.rst#L85-L88

Reply:
Yep, sounds good! Let me add a TODO in https://github.com/apache/airflow/pull/39585